composer.json

```diff
@@ -17,11 +17,11 @@
         {
             "name": "Brian Nesbitt",
             "email": "brian@nesbot.com",
-            "homepage": "http://nesbot.com"
+            "homepage": "https://nesbot.com"
```

Comment:
This website does not have a valid HTTPS certificate (it maybe never did).

Hello @briannesbitt, please check if you want to update your website link and give HTTPS link if you have. Thank you.

Reply:
Makes sense @kylekatarnls